Library/Formula/flex.rb

```diff
@@ -15,6 +15,8 @@ class Flex < Formula
   depends_on "gettext"
 
   def install
+    # 2.5.38+ build fix adapted from http://patchwork.openembedded.org/patch/68029/
+    ENV["LDFLAGS"] = "#{ENV["LDFLAGS"]} -all-static"
```

Comment:
Yup.

—
Sent from Mailbox

On Thu, Jan 8, 2015 at 5:08 AM, Mike McQuaid <notifications@github.com>
wrote:

>> @@ -15,6 +15,8 @@ class Flex < Formula
>>    depends_on "gettext"
>>  
>>    def install
>> +    # 2.5.38+ build fix adapted from http://patchwork.openembedded.org/patch/68029/
>> +    ENV["LDFLAGS"] = "#{ENV["LDFLAGS"]} -all-static"
> Ok, can we use that instead of passing `LDFLAGS`?
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/Homebrew/homebrew/pull/35538/files#r22650401